spring-xd-dirt/src/main/java/org/springframework/xd/dirt/integration/bus/serializer/kryo/PojoCodec.java

```diff
@@ -41,12 +40,4 @@ protected Object doDeserialize(Kryo kryo, Input input, Class<? extends Object> t
 		kryo.register(type);
```

Comment:
Now that the Kryo instances are pooled and shared across multiple invocations, does it matter that we register types on-demand, in arbitrary order? The reason I ask is this comment in the Kryo#register(..) JavaDoc:

````
* Because the ID assigned is affected by the IDs registered before it, the order classes are registered is important when
* using this method. The order must be the same at deserialization as it was for serialization. */
````